recipe/meta.yaml

```diff
@@ -31,8 +36,13 @@ source:
     folder: plugins/access-control
 
 build:
-  number: 0
+  number: 1
   skip: True   # [unix]
+  string: h{{ PKG_HASH }}_log_{{ PKG_BUILDNUM }}  # [nsis_variant == 'log_enabled']
+  track_features:
+    - nsis_log_enabled  # [nsis_variant == 'log_enabled']
+
+requirements:
 
 test:
   commands:
```

Comment:
Is it possible to check for a generated log file as part of the test for the log_enabled variant?

Reply:
Not that I know of because the installers need to be built with that NSIS variant first. I confirmed in a separate repository though that installers built with that variant do produce an output though: https://github.com/anaconda-distribution/installers/actions/runs/4611007765/jobs/8150162202#step:6:3462

The test would fail if there wasn't a log file:

```
@pytest.mark.skipif(os.getenv("NSIS_USING_LOG_BUILD") != "1", reason="Logs not enabled")
def test_uninstaller_output(destination_path):
    log_file = destination_path / "install.log"
    assert log_file.exists()
    lines = log_file.read_text().split("\n")
    assert all(":error:" not in line for line in lines)
```